vsphere/datadog_checks/vsphere/data/conf.yaml.example

```diff
@@ -82,49 +60,63 @@ instances:
     #
     # ssl_capath: "<DIRECTORY_PATH>"
 
-    ## @param host_include_only_regex - string - optional
-    ## Use a regex like this if you want only the check
-    ## to fetch metrics for these ESXi hosts and the VMs
-    ## running on it
-    #
-    # host_include_only_regex: <REGEX>
-
-    ## @param vm_include_only_regex - string - optional
-    ## Use a regex to include only the VMs that are
-    ## matching this pattern.
-    #
-    # vm_include_only_regex: <REGEX>
-
-    ## @param include_only_marked - boolean - optional - default: false
-    ## Set to true if you'd like to only collect metrics on vSphere VMs which
-    ## are marked by a custom field with the value 'DatadogMonitored'
-    ## To set this custom field with PowerCLI, use the follow command:
-    ##   Get-VM <MyVMName> | Set-CustomField -Name "DatadogMonitored" -Value "DatadogMonitored"
-    #
-    # include_only_marked: false
-
-    ## @param collection_level - integer - optional - default: 1
-    ## A number between 1 and 4 to specify how many metrics will be sent
-    ## 1: Only basic metrics - 4: every metric available.
-    ## Warning: Depending on the size of the vSphere environment, metric collection can be slow,
-    ## very CPU intensive and put pressure on the vCenter Server
-    #
-    # collection_level: 1
-
-    ## @param collect_historical_only - boolean - optional - default: false
-    ## By default the check collects metrics from realtime resources (hosts and VMs).
-    ## Set this to true to only collect metrics from historical resources
-    ## (cluster, datastore and datacenter).
-    ## In order to collect metrics from those two types of resources, create two
-    ## 'instances' in this config file pointing to the same vcenter server, one with this flag
-    ## to false, the other with this flag to true.
-    ## The reason for this requirement is that collecting historical resources decreases
-    ## the performance of the check; having two instances allows realtime metrics collection
-    ## not to be slowed down by the collection of historical metrics.
-    ## Note: An instance with this flag set to true does not submit any service check, events or host tags.
-    ##       It is designed to be used only when next to another instance of the check.
-    #
-    # collect_historical_only: false
+    ## @param resource_filters - list of objects - optional - default: no filter
```

Comment:
The resource_filters make sense and great :)

Is there a need to provide both `blacklist_patterns` and `whitelist_pattern` for vSphere use case?

Reply:
Good question, I think this is fine for now as users should be able to do whatever they need with a list of regexes. Future will tell if we need to add blacklists, this can be done later.